public/models/feathers-client.js

```diff
@@ -1,8 +1,13 @@
 import feathers from 'feathers/client';
-import io from 'steal-socket.io';
+//import io from 'steal-socket.io';
+import io from 'socket.io-client/dist/socket.io';
 import socketio from 'feathers-socketio/client';
 import auth from 'feathers-authentication-client';
 import hooks from 'feathers-hooks';
+import "@fixture-socket-hook";
```

Comment:
To make sure feathers uses the "mocked" socket in test (so can-fixture-socket works), it imports `"@fixture-socket-hook"` which is mapped to `@empty` in development, and `public/models/fixtures/fixture-socket-server.js` in test.